Mathlib/Data/Nat/Bitwise.lean

```diff
@@ -48,6 +48,17 @@ set_option linter.deprecated false
 section
 variable {f : Bool → Bool → Bool}
 
+lemma succ_testBit_zero (n : ℕ) : (n + 1).testBit 0 = !n.testBit 0 := by
+  apply n.bitCasesOn
+  rcases; simp [testBit_bit_zero, -testBit_zero]
```

Comment:
What's this `rcases` doing? I don't think I've seen it without an argument before

Reply:
It seems like it didn't do anything, so I've removed it.